tests/acceptance/features/coreApiWebdavOperations/listFiles.feature

```diff
@@ -72,35 +72,6 @@ Feature: list files
       | dav-path-version |
       | spaces           |
 
-  @depthInfinityPropfindEnabled
-  Scenario Outline: get the list of resources in the root folder with depth infinity
```

Comment:
will be covered in new test suite with all dav paths (`old`, `new` and `spaces`)